core/services/wifi/WifiManager.py

```diff
@@ -139,7 +139,10 @@ async def add_network(self, credentials: WifiCredentials) -> int:
 
             network_number = int(data)
             await self.wpa.send_command_set_network(network_number, "ssid", f'"{credentials.ssid}"')
-            await self.wpa.send_command_set_network(network_number, "psk", f'"{credentials.password}"')
+            if not credentials.password:
```

Comment:
What happens if the user just want to connect in a network that he has previous configured and don't want to set the password (since we have it stored) ?

Reply:
This case is handled by `connect_to_network` method. This method here is just for adding new ones.